schemas/v3.0/schema.yaml

```diff
@@ -1,6 +1,6 @@
-id: https://spec.openapis.org/oas/3.0/schema/2021-09-28
```

Comment:
I think part of the problem with out publishing process is has to do with this date.  I'm fine with manually setting it when we push it out, but I think there was some expectation to set it automatically.  Perhaps documented in #3715

Reply:
The file in git should still have an `id`/`$id` of some kind though.. perhaps something with "in_progress" or "latest" or something else non-dateish to indicate that it's not reflecting an officially-published version. Otherwise, it's yet one more barrier to tool vendors (on top of converting yaml to json) to make use of the interim file for development and testing.

Reply:
@karenetheridge I'd lean towards something like "in_progress" that discourages casual use.

There really shouldn't be interim files if our publication process is running smoothly.  It might be different for pre-publication releases, but we should handle that separately as that's not a process we've defined at all yet.

In theory, every time we change a published schema, we just re-publish it immediately.  If we ever have a reason _not_ to do that, then we probably also don't want people to use the schemas out of `main`.  We really only want the published ones to be usable, I think (again, excluding work-in-progress versions where we just haven't defined a process).

I do agree that if we _do_ want people to use things directly from `main`, then they need `$id`/`id` values that are somehow reasonable, and not identical to any published version.

Reply:
The widely used [@apidevtools/openapi-schemas](https://www.npmjs.com/package/@apidevtools/openapi-schemas) package (1.7 million weekly downloads) seems to pull the OpenAPI schemas directly from `main`:

> This package contains **[the official JSON Schemas](https://github.com/OAI/OpenAPI-Specification/tree/master/schemas)** for every version of Swagger/OpenAPI Specification

And there may be other uses out there we don't know of.

Better have up-to-date YAML and JSON files with the most current identifier in the `main` branch.

Reply:
Well, then let's replace the date with "in-progress" and see what gets published to the NPM package registry 😎

Reply:
@ralfhandl I wouldn't object to trying to coordinate with them a bit.  This is part of my (myriad) frustration with the schemas- we don't exactly make it easy for people who want to do something like package and distribute to do the right thing.  It was only a couple of months ago that I even realized that "published" meant "put on the GitHub pages site" since we didn't even link to them on the index page of that site.   It's far easier to find the "wrong" files than the "right" ones.